msteams-platform/bots/how-to/conversations/notification-bot-in-teams.md

```diff
@@ -48,15 +48,18 @@ When you send notifications, TeamsFx SDK creates a new conversation from the sel
 
 ```TypeScript
 
-   // list all installation targets
-for (const target of await bot.notification.installations()) {
-    // call Bot Framework's adapter.continueConversation()
-    await target.adapter.continueConversation(target.conversationReference, async (context) => {
-        // your own bot logic
-        await context...
-    });
+// list all installation targets
+for (const target of await notificationApp.notification.installations()) {
+    // call Bot Framework's adapter.continueConversationAsync()
+    await target.adapter.continueConversationAsync(
+        target.botAppId,
+        target.conversationReference,
+        async (context) => {
+            // your own bot logic
+            await context...
+        }
+    );
 }
-```
```

Comment:
Add this line back? Removing this line breaks the code snippet
<img width="524" alt="image" src="https://github.com/MicrosoftDocs/msteams-docs/assets/7642967/7e908eb4-7014-432a-a7c6-8574fa38b96c">